ci/tasks/run-integration-windows.sh

```diff
@@ -88,6 +88,10 @@ export BOSH_AZURE_WINDOWS_LIGHT_STEMCELL_VERSION=$(ruby -r yaml -r json -e '
     end
   end' < /tmp/stemcell.MF)
 
+# Accept legal terms
+image_urn="pivotal:bosh-windows-server:${BOSH_AZURE_WINDOWS_LIGHT_STEMCELL_SKU}:${BOSH_AZURE_WINDOWS_LIGHT_STEMCELL_VERSION}"
+az vm image accept-terms --urn ${image_urn}
```

Comment:
better to turn on the --debug option for debugging if any error occured. maybe --verbose is even acceptable.

Reply:
Currently, I think there's no need to show all debug logs.

Reply:
then i think we should log down the image_urn value. because the variables are fetched by scripts.
it will help us diagnostic why the accept failed or something like that.

Reply:
sure. will log down the image_urn value.
BTW, the output is:
```
{
  "accepted": true,
  "id": "/subscriptions/c4528d9e-c99a-48bb-b12d-fde2176a43b8/providers/Microsoft.MarketplaceOrdering/offerTypes/Microsoft.MarketplaceOrdering/offertypes/publishers/pivotal/offers/bosh-windows-server/plans/2012r2-20plus/agreements/current",
  "licenseTextLink": "https://storelegalterms.blob.core.windows.net/legalterms/3E5ED_legalterms_PIVOTAL%253a24BOSH%253a2DWINDOWS%253a2DSERVER%253a242012R2%253a2D20PLUS%253a247JYMYE2YASU5AEQC27QEJZXNN5KSZJROSTALRTIQMMQ4ZJXP5ODZT4NCNPDHMG6Y5VLPO54FLZ7QP275PPW5YB36TH6K6EMUL3JPY7Y.txt",
  "name": "2012r2-20plus",
  "plan": "2012r2-20plus",
  "privacyPolicyLink": "https://pivotal.io/privacy-policy",
  "product": "bosh-windows-server",
  "publisher": "pivotal",
  "retrieveDatetime": "2018-08-09T14:17:59.5415172Z",
  "signature": "DWDJLAGOXAIZC3VJJOIJ6JKRENDUT6OUHW7M5RAN7SHM5I6DZZDZLM5BTD6W7FMCPK4ETICLRQG3ZKUUCAFYL2UYWLKRAWMLDEUBDPQ",
  "type": "Microsoft.MarketplaceOrdering/offertypes"
}
```